src/activities/menu/Menu.qml

```diff
@@ -459,6 +500,136 @@ ActivityBase {
 
         }
 
+
+        Rectangle
```

Comment:
put the accolade in the same line as the item: Rectangle {
Same for below with the TextInput and VirtualKeyboard